src/offchain/eth.ts

```diff
@@ -91,7 +91,7 @@ export const sendEthErc20OffchainTransaction = async (testnet: boolean, body: Tr
     const vc = await getVirtualCurrencyByName(account.currency);
     const {txData, gasLimit} = await prepareEthErc20SignedOffchainTransaction(amount, fromPriv, address, web3, vc.erc20Address as string, gasPrice, nonce);
     // @ts-ignore
-    withdrawal.fee = new BigNumber(web3.utils.fromWei(new BigNumber(gasLimit).multipliedBy(gasPrice).toString(), 'ether')).toString();
+    withdrawal.fee = new BigNumber(web3.utils.fromWei(new BigNumber(gasLimit || body.gasLimit).multipliedBy(gasPrice).toString(), 'ether')).toString();
```

Comment:
should be the opposite. body.gasLimit || gasLimit. gasLimit is still present, you want to replace it from body, if present.